Golem.Tests/GolemTests.cs

```diff
@@ -96,21 +109,24 @@ public async Task LoadBinaryStartAndStop_VerifyStatusAsync()
             Assert.Equal(GolemStatus.Off, status);
         }
 
-        [Fact]
+        // [Fact]
```

Comment:
Remaining yagna process (which is doing its shutdown steps) breaks other tests (? - need to fix it)

Reply:
Should be resolved here https://github.com/golemfactory/ya-runtime-ai/issues/37